docs/sources/dashboards/assess-dashboard-usage/index.md

```diff
@@ -21,7 +21,7 @@ weight: 200
 
 Usage insights enables you to have a better understanding of how your Grafana instance is used.
 
-> **Note:** Available in [Grafana Enterprise]({{< relref "../" >}}) and [Grafana Cloud Pro and Advanced]({{< ref "/docs/grafana-cloud" >}}).
+> **Note:** Available in [Grafana Enterprise]({{< relref "../" >}}) and [Grafana Cloud Pro and Advanced]({{< ref "/docs/grafana-cloud" >}}). Grafana Cloud provides additional features via the Usage Insights dashboards. Read more [here](TODO).
```

Comment:
```suggestion
> **Note:** Available in [Grafana Enterprise]({{< relref "../" >}}) and [Grafana Cloud Pro and Advanced]({{< ref LINK SOMEWHERE ELSE HERE? >}}). Grafana Cloud provides additional features via the Usage Insights dashboards. Read more [here](TODO).
```

Reply:
NIT: "Grafana Cloud provides additional features via the Usage Insights dashboards"

Maybe more like "The Usage Insights dashboards available within GrafanaCloud have some additional features - read more _here_"

Reply:
I don't understand what you mean with the `LINK SOMEWHERE ELSE HERE?`. Not sure where else that would link?

Reply:
Sorry, I think I explained it better while we were pairing. 
The link text talks about Pro and Advanced specifically. If we could link somewhere more specific, I think that would be an improvement. Not a big deal though, and just a suggestion - there's probably nowhere better to link to right now.